### PR TITLE
Add support for testing GW routers.

### PR DESCRIPTION
**Pull request by dceara**

### CODE REVIEW

**Comment by dceara** [comment]
@hzhou8, when you have time it would be great if you could have a look at these changes, they upstream all the remaining code we were using in our downstream tests.

Thanks,
Dumitru

File: rally_ovs/plugins/ovs/ovnclients.py [hunk 1]
```diff
@@ -44,9 +44,9 @@ def _restart_daemon(self, nbctld_config):
         self._stop_daemon()
         return self._start_daemon(nbctld_config)
 
-    def _get_gw_ip(self, network_cidr):
-        # Use the last IP in the CIDR as gateway IP.
-        return netaddr.IPAddress(network_cidr.last - 1)
+    def _get_gw_ip(self, network_cidr, offset=1):
+        # Use the last IP (+ offset) in the CIDR as gateway IP.
```
Comment (hzhou8) [inline rally_ovs/plugins/ovs/ovnclients.py]: This comment is confusing. Did you mean (- offset)?

**Review by hzhou8 — CHANGES_REQUESTED** [review]

**Review by hzhou8 — APPROVED** [review]

**Comment by hzhou8** [comment]
@dceara LGTM except a very minor comment.

File: rally_ovs/plugins/ovs/ovnclients.py (hunk 1 above)
Comment (dceara) [reply rally_ovs/plugins/ovs/ovnclients.py]: Right, thanks for spotting it, i'll fix it.

**Comment by dceara** [comment]
@hzhou8 thanks for the review, I'm merging the PR.

**Pull request merged**